yt_dlp/extractor/youtube.py

```diff
@@ -3736,6 +3740,8 @@ def _extract_formats_and_subtitles(self, streaming_data, video_id, player_url, l
             if is_damaged:
                 self.report_warning(
                     f'{video_id}: Some formats are possibly damaged. They will be deprioritized', only_once=True)
+
+            client_name = (traverse_obj(query, ('c', 0, {str})) or fmt.get(STRAMING_DATA_CLIENT_NAME) or '')[0:3].upper()
```

Comment:
With 3 chars, we can't tell `web_embedded` etc apart. Maybe use thw whole string?

Reply:
The reason behind 3 characters is that it's narrow enough not to cause break lines in my console and it's long enough to identify which device is problematic, at first glance
And if we need the whole string, it should be done by editing code as needed